boa_tester/src/edition.rs

```diff
@@ -0,0 +1,332 @@
+use std::fmt::Display;
```

Comment:
We should add a bit of module-level documentation, and a bit of documentation in all structs, constants and methods.